browser/src/slideshow/PresenterConsole.js

```diff
@@ -170,6 +170,7 @@ class PresenterConsole {
 				img = this._proxyPresenter.document.createElement('img');
 				img.id = `preview-slide-${index}`;
 				img.src = document.querySelector('meta[name="previewImg"]').content;
+				img.alt = _(`Preview Slide ${index + 1}`);
```

Comment:
We need to provide constant string for translation (not variant with different numbers :) ).
So please use `_('Preview Slide {1}')` and `.replace('{1}', (index + 1))`; 

example: https://github.com/CollaboraOnline/online/blob/0edb0befe43d200ff471ed5f813b86fe538f2f74/browser/src/slideshow/PresenterConsole.js#L1217